dts/bindings/bluetooth/nordic,nrf-dfe-ant.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+#
+# Note:
+#  All dfegpio[n]-pin properties have required property set to false. It
+#  depends on end user which Radio PSEL_GPIO[n] registers will be used to drive
+#  antenna switches.
+#  Nevertheless there is a requirement that at least two antennae are available.
+#  That means at least one dfegpio[n]-pin must be provided, to give Radio
+#  possibility to switch between these two antennae.
+
+description: |
+    Nordic DFE antennae matrix description
+
+compatible: "nordic,nrf-dfe-ant"
```

Comment:
These properties seem to be going straight into RADIO peripheral registers.

I might be wrong, but it really looks like we should be making a `nordic,nrf-radio` compatible instead.

Reply:
That might be better, yes.

Reply:
Shouldn't it go to radio directory if name of binding is changed to `nordic,nrf-radio`?

Reply:
It should go into `dts/bindings/net/wireless` since we now have that from #30788.  That directory is where Linux puts radio hardware.

Reply:
Moved to suggested location.